<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>55</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Add Window&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Window</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_WIndow">
    <property name="title">
     <string>Open WIndow</string>
    </property>
    <addaction name="actionWindow_1"/>
    <addaction name="actionWindow_2"/>
    <addaction name="actionWindow_3"/>
   </widget>
   <widget class="QMenu" name="menuClose_Window">
    <property name="title">
     <string>Close Window</string>
    </property>
    <addaction name="actionClose_1"/>
    <addaction name="actionClose_2"/>
    <addaction name="actionClose_3"/>
   </widget>
   <addaction name="menuOpen_WIndow"/>
   <addaction name="menuClose_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWindow_1">
   <property name="text">
    <string>Window 1</string>
   </property>
  </action>
  <action name="actionWindow_2">
   <property name="text">
    <string>Window 2</string>
   </property>
  </action>
  <action name="actionWindow_3">
   <property name="text">
    <string>Window 3</string>
   </property>
  </action>
  <action name="actionClose_1">
   <property name="text">
    <string>Close #1</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close #2</string>
   </property>
  </action>
  <action name="actionClose_3">
   <property name="text">
    <string>Close #3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
